reference/state.md

```diff
@@ -140,3 +140,17 @@ $ pulumi stack import --file my-app-production.checkpoint.json # import the new
 In addition, if you have any encrypted configuration in your stack, you'll need to re-run
 `pulumi config set --secret <key> <value>` because pulumi.com uses a different key to encrypt your secrets than the
 local endpoint.
+
+### Secrets Encryption
+
+When a secret configuration value is set (by passing `--secret` to `pulumi config set`) or created inside your program
```

Comment:
The wording here is a little confusing - "secret configuration values" are not "created inside your program".  

I think this means:

> When a secret value is provided via secret configuration (by passing `--secret` to `pulumi config set`) or created inside your program ...